app/src/main/java/com/github/polypoly/app/base/game/location/LocationProperty.kt

```diff
@@ -17,7 +17,7 @@ data class Zone(
     An enum class representing the different levels of a [LocationProperty].
  */
 enum class PropertyLevel {
-    LEVEL_0, LEVEL_1, LEVEL_2, LEVEL_3, LEVEL_MAX
+    LEVEL_0, LEVEL_1, LEVEL_2, LEVEL_MAX
```

Comment:
We will have to think about replacing them with our real level names. Why did we delete a level by the way?

Reply:
Reason 1, convenience and making the stars fit on the bitmap, I can make 3 stars fit ish (technically only 2 fit but conveniently 3 shows a full star and 2 star halves on either end)
Reason 2, we will never reach 4 levels anyway.